src/prefect/agent/kubernetes/job_spec.yaml

```diff
@@ -17,7 +17,7 @@ spec:
           image: prefecthq/prefect:latest
           imagePullPolicy: IfNotPresent
           command: ["/bin/sh", "-c"]
-          args: ["prefect execute cloud-flow"]
+          args: ["prefect execute flow-run"]
```

Comment:
Are there any ways for users to override this initial image with something that might be have older version of Prefect?

Reply:
We don't expose a setting/kwarg for it but if something like #3046 is adopted then there is a chance it could happen

Reply:
yea, I guess let's just remember this when we go to refactor agents / environments